lib/cf/broker/manifest.yml

```diff
@@ -7,11 +7,11 @@ applications:
   disk_quota: 512M
   env:
     CONF: default
-    DEBUG: e-abacus-*
+    DEBUG: metering-*,e-metering-*
```

Comment:
We should rename the debug string to `abacus-broker` to match the rest of the abacus modules.